### ISSUE TITLE
Expose Module Usage Behaviors as Ember Actions

### ISSUE THREAD

**Issue by Panman8201** [Semantic-Org/Semantic-UI-Ember#80]
It would be nice to have each modules usage behaviors (I think that's what Semantic UI calls them) be exposed as ember component actions. For example; a sidebar "toggle" behavior/method:

```javascript
actions: {
  toggle() {
    this.$().sidebar('toggle');
  }
}
```

Then code that has access to the sidebar instance (parent/child component, controller, whatever) can call upon the action instead of using the jQuery plugin. Seems more Ember'ish anyway.

```javascript
sidebarComponent.send('toggle');
```

I'd be willing to work on a PR if I could get some guidance on how you'd like to see it implemented.

**Comment by Panman8201** [comment]
I just wanted to note my specific use case where I've found this helpful. Inside the `{{ui-sidebar}}` block I have a menu of `{{link-to}}`'s which (with the addition of the [ember-link-action](https://www.npmjs.com/package/ember-link-action) addon) invoke the "hide" action, hiding the sidebar while transitioning to the new route.

```hbs
{{#ui-sidebar class="inverted vertical menu"}}
  {{link-to "Foo" "foo" class="item" invokeAction="hide"}}
  {{link-to "Bar" "bar" class="item" invokeAction="hide"}}
  {{link-to "Baz" "baz" class="item" invokeAction="hide"}}
{{/ui-sidebar}}
```

**Comment by brendan-rius** [comment]
I didnt see your issue before, and I just opened a new one that might (at least a bit) correspond to your problem: https://github.com/Semantic-Org/Semantic-UI-Ember/issues/83

**Comment by aaronbhansen** [comment]
Thanks for the example, that makes more sense. We are looking to get an initial 1.0 release out that uses mutable properties, and then move to a 2.0 release that applies data down actions up methodology. Every semantic ember component has a method called execute on it, that invokes the semantic module directly. We could expose that as a composable action, that could be called as a block parameter to nested children. I'll think about this a little more, but I think you bring up a good usage example of where to head in the next release.

**Comment by aaronbhansen** [comment]
Now available in 2.0 beta as a composable action
